ironfish/src/testUtilities/helpers/serializable.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { NetworkMessage } from '../../network/messages/networkMessage'
+import { RpcNetworkMessage } from '../../network/messages/rpcNetworkMessage'
+import { WorkerMessage } from '../../workerPool/tasks/workerMessage'
+
+export function serializePayload(
```

Comment:
Yeah, I could see how this would be confusing, maybe `renderPayload`?

Reply:
Fair point - I was considering renaming `serializePayload` on the messages to `writePayload` or something, since its more inline with what it actually does. It's how we name our read/write functions in the Rust code, too. What do you think?

Reply:
I'm not necessarily opposed to that, but it might be a more involved rename since I wonder at that point if we should rename serialize/deserialize to write/read.

Maybe `serializePayloadToBuffer` for the test helper instead?